static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
         &:before {
             .cf-icon();
-            content: "\e402";
+            content: "\e406";
```

Comment:
We should explore extending the `.cf-icon()` mixin to include an argument that passes in the keyword version of icons instead of remembering/looking up unicodes.